client/src/components/Create.jsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Box,
+  TextField,
+  Stack,
+  Button,
+  Card,
+  CardHeader,
+  CardContent,
+  Typography,
+} from "@mui/material";
+
+const styles = {
```

Comment:
If you are going to use CSS styling, it would be best to use dedicated css files for these

Reply:
Alternatively  you could look into styled components. https://styled-components.com/. You could take a look at either Kevin or Emily's prs for a reference.